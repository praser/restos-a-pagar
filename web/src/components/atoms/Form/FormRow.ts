import styled from 'styled-components';

const FormRow = styled.div`
  display: flex;
  flex-wrap: wrap;
  margin-right: -5px;
  margin-left: -5px;
`;

export default FormRow;
