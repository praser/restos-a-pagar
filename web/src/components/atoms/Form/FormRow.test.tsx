import React from 'react';

import renderer from 'react-test-renderer';

import FormRow from './FormRow';

describe('FormRow atom component', () => {
  it('is expected to render the component correctly', () => {
    const formRow = renderer.create(<FormRow />);
    expect(formRow).toMatchSnapshot();
  });
});
