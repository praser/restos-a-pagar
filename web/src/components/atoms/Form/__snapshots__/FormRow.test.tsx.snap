// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormRow atom component is expected to render the component correctly 1`] = `
<div
  className="sc-bdfBwQ ifChW"
/>
`;
